<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateWalletWindow1</class>
 <widget class="QDialog" name="CreateWalletWindow1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Wallet</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Wallet Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leWalletName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Mnemonic Language</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbMnemonicLang">
         <item>
          <property name="text">
           <string>English (Default)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>简体中文</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>日本語</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Français</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Español</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Mnemonic Length</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbMnemonicLen">
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>21</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnNext">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>CreateWalletWindow1</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
